src/seedu/addressbook/storage/jaxb/AdaptedPerson.java

```diff
@@ -60,7 +60,7 @@ public AdaptedPerson(ReadOnlyPerson source) {
 
         address = new AdaptedContactDetail();
         address.isPrivate = source.getAddress().isPrivate();
-        address.value = source.getAddress().value;
+        address.value = source.getAddress().toString();
```

Comment:
Hello Akshay,

Thanks for your comments! Concerning this comment, the issue I'm facing is that getAddress() return type is Address, not String. Also, other methods such as getEmail() returns Email, getPhone() returns Phone. As a result, by changing the return type of getAddress() to be String, it isn't consistent with other methods.

Are there other ways about this? :)